helm/idol-mediaserver/resources/mediaserver.cfg

```
@@ -0,0 +1,55 @@
+// patching config file
+// idol_ssl.cfg is written by run_idol.sh with SSL settings if required by IDOL_SSL env variable
+// original.mediaserver.cfg is the config file as contained in the standard component zip
+
+// modified config sections
+
+< "/mediaserver/cfg/idol_ssl.cfg" [SSLSettings]
+
+[License] < "/mediaserver/cfg/original.mediaserver.cfg" [License]
+LicenseServerHost={{ .Values.licenseServerHostname }}
+LicenseServerACIPort={{ .Values.licenseServerPort | int }}
+
+[Service] < "/mediaserver/cfg/original.mediaserver.cfg" [Service]
+Access-Control-Allow-Origin=*
+ServicePort={{ .Values.servicePort | int }}
+
+[Server] < "/mediaserver/cfg/original.mediaserver.cfg" [Server]
+< "/mediaserver/cfg/idol_ssl.cfg" [EnableSSL] SSLConfig
+
```

Need to ensure `aciPort` value gets used
```suggestion
[Server] < "/mediaserver/cfg/original.mediaserver.cfg" [Server]
< "/mediaserver/cfg/idol_ssl.cfg" [EnableSSL] SSLConfig
Port={{ .Values.aciPort | int }}
```